store/store.go

```diff
@@ -172,19 +165,38 @@ func (s *Store) createFile(
 }
 
 func (s *Store) ensureHeightLink(datahash share.DataHash, height uint64) error {
-	path := s.hashToPath(datahash)
 	// create hard link with height as name
-	linkPath := s.heightToPath(height)
-	err := os.Link(path, linkPath)
+	pathOds := s.hashToPath(datahash, odsFileExt)
+	linktoOds := s.heightToPath(height, odsFileExt)
+	pathQ4 := s.hashToPath(datahash, q4FileExt)
+	linktoQ4 := s.heightToPath(height, q4FileExt)
+	return errors.Join(
+		link(pathOds, linktoOds),
+		link(pathQ4, linktoQ4),
+	)
+}
+
+func (s *Store) ensureEmptyFile() error {
+	roots, err := eds.EmptyAccessor.AxisRoots(context.Background())
+	if err != nil {
+		panic(fmt.Sprintf("error getting roots of an empty accessor: %s", err))
+	}
```

Comment:
why not simple share.EmptyEDSRoots?

Reply:
It felt wrong to request the same thing from two different pkgs.

Reply:
[39beb05](https://github.com/celestiaorg/celestia-node/pull/3643/commits/39beb053157377caaa21578e904c4f2d5cc7fe08)